actions.yml

```diff
@@ -130,3 +130,14 @@ github/codeql-action/analyze:
     contents: read
     pull-requests: read
     security-events: write
+ridedott/merge-me-action:
+  permissions:
+    contents: write
+shogo82148/actions-check-permissions:
+# https://github.com/saucelabs/sauce-connect-action/issues/24
+saucelabs/sauce-connect-action:
+  permissions:
+    contents: write
```

Comment:
Is this `contents`  permissions required?
In other words,  Does saucelabs/sauce-connect-action push to GitHub repository?

Reply:
saucelabs/sauce-connect-action requires an access token, so probably we should use GitHub secrets. 
I think it requires `contents: write` permission, is that wrong?

Reply:
For example, workflows triggered by `push` of dependabot have only read permission, so they cannot access secrets.

https://github.blog/changelog/2021-02-19-github-actions-workflows-triggered-by-dependabot-prs-will-run-with-read-only-permissions/

Reply:
https://github.com/saucelabs/sauce-connect-action#accesskey
This Actions requires SaurcesLab's accesskey, not `secrets.GITHUB_TOKEN`.
We do not require any `permissiosn` when use `${ secrets.FOO_SECRET_KEY }`.

This `permissions` value is for `secrets.GITHUB_TOKEN`.
https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/

I think that this actions does not require any `permissions` for `secrets.GITHUB_TOKEN`.

Reply:
Sorry, I was wrong.
I experimented and found that access to secrets depends on whether it was forked or not, not on `contents: write` permissions.